src/styles/thumbnail.css

```
@@ -0,0 +1,72 @@
+  user-select: none;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.item-desc {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  font-family: "Inter", sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)